docs/stake-pool-guide/stake-pool-networking.md

```diff
@@ -0,0 +1,19 @@
+---
+id: stake-pool-networking
+title: Understanding the Relay and Block Producer topology
+sidebar_label: Stake Pool Networking
+description: Understanding the Relay and Block Producer topology
+image: ../img/og-developer-portal.png
+---
+Before we start with the stake pool installation and configuration, it is essential to understand the logical topology of stake pools. Typically every stake pool has one block producer and at least one relay node. To ensure fast propogation of blocks the pool produces, it is recommended to have relays in different geographical locations. To secure the block producer from the internet it should only connect to its own relays, or relays you trust. The relays then connect to the rest of the Cardano network. To minimise the risk of height battles, it is recommended to regularly monitor the propogation time of the relays. We will talk more on this topic in our pool monitoring section later.
+
+In the example below, one block producer is connected to three relays and they are connected to other relays of the Cardano network.
+
+![img](../../static/img/stake-pool-guide/stake-pool-network.jpg)
+
+For the operation of a stake pool, some additional nodes in the topology might be useful. For example, a back-up node for block producer is helpful in case the main block producer has an issue e.g. during an upgrade. Depending on the number of stake pools that are being managed, a monitoring node can be used to monitor them and generate alarms. Stake pool operators also need a [air-gapped system](/docs/get-started/air-gap) to store the pool cold keys. It is also recommended to have a hardware wallet for storing the pool pledge.
```

Comment:
Before committing this disputable (and often disputed) sentence:

> It is also recommended to have a hardware wallet for storing the pool pledge.

... editorial standards require that you provide a reference to exactly what recommendation you are talking about: https://en.wikipedia.org/wiki/Weasel_word

Reply:
how about re-framing to "It is good practice...."?

Reply:
It is _one_ practice: _one_ possible approach to avoid exposing stake pool keys.  If you want to say it's a "good" practice then just either:

1. include a source for that statement, or
2. explain yourself why the hardware wallets provide adequate security for the stake pool keys: though according to the outline you helped develop, this question is better answered on the Security page(s) rather than the Networking page.

Another way to avoid that subjective statement would be to leave the security partitioned devices out of the Network page altogether... because they are, by definition, _not_ part of the network.

Reply:
You are right, so have excluded the HW Wallet from the topology, but wish to still mention this option so would formulate like - Some pool operators use a hardware wallet for storing the pool pledge.
fine?